DataStructures.Tests/SortedListTests.cs

```diff
@@ -1,6 +1,7 @@
 using System.Collections.Generic;
 using System.Linq;
 using NUnit.Framework;
+using NUnit.Framework.Legacy;
```

Comment:
Do you need this using?

Reply:
It was necessary because of the old NUnit assert that I was using but after the change it wont be necessary anymore